### PR TITLE
映射文档 No. 34

### PR DESCRIPTION
**Pull request by cccbbbttt**
update md

### CODE REVIEW

**Comment by cccbbbttt** [comment]
您好，请您check一下PR@Tomoko-hjf

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter_.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+两者功能一致且参数用法一致，仅参数名不一致，具体如下：
+
+### 参数差异
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| dim           | axis        | 表示在哪一个维度 scatter ，仅参数名不一致。 |
+| index         | indices     | 表示输入的索引张量，仅参数名不一致。                   |
+| src           | values      | 表示需要插入的值，仅参数名不一致。                   |
+| reduce        | reduce      | 表示对输出 Tensor 的计算方式，与 PyTorch 默认值不同+ Paddle 保持默认即可。  |
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter_.md]: 这里如果不指定的话都是直接覆盖吧，所以只写 `表示对输出 Tensor 的计算方式。`就可以
image

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter_add_.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+两者功能一致且参数用法一致，仅参数名不一致，具体如下：
+
+### 参数差异
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| dim           | axis        | 表示在哪一个维度 scatter ，仅参数名不一致。 |
+| index         | indices     | 表示输入的索引张量，仅参数名不一致。                   |
+| src           | values      | 表示需要插入的值，仅参数名不一致。                   |
+| -             | reduce      | 表示对输出 Tensor 的计算方式， PyTorch 无此参数+ Paddle 应设置为 add 。  |
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter_add_.md]: 把`+`号换成`，`号吧~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.scatter_add.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+```python
+paddle.put_along_axis(arr,
+                      indices,
+                      values,
+                      axis,
+                      reduce='assign')
+```
+
+两者功能一致且参数用法一致，仅参数名不同，同时 Paddle 支持更多其他参数，具体如下：
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.scatter_add.md]: 这句话直接写 `Paddle 相比 Pytorch 支持更多其他参数，具体如下：`就可以

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.scatter_add.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+## [ 仅 paddle 参数更多 ]torch.scatter_add
+
+### [torch.scatter_add](https://pytorch.org/docs/1.13/generated/torch.scatter_add.html#torch.scatter_add)
+
+```python
+torch.scatter(input,
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.scatter_add.md]: api签名少了`_add`

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.scatter_add.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+两者功能一致且参数用法一致，仅参数名不同，同时 Paddle 支持更多其他参数，具体如下：
+
+### 参数差异
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| input        | arr         | 表示输入 Tensor ，仅参数名不一致。                                     |
+| dim          | axis        | 表示在哪一个维度 scatter ，仅参数名不一致。                             |
+| index        | indices     | 表示输入的索引张量，仅参数名不一致。                                    |
+| src          | values      | 表示需要插入的值，仅参数名不一致。                                      |
+| -            | reduce      | 表示对输出 Tensor 的计算方式， PyTorch 无此参数+ Paddle 应设置此参数为 add 。|
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.scatter_add.md]: +号改为逗号就可以~

**Review by Fancy-hjyp — COMMENTED** [review]
转写示例如果使用的是具体的数值，需要用torch.tensor()或paddle.to_tensor()处理一下，不然运行不能通过，尽量给用户正确的示例~

**Comment by Fancy-hjyp** [comment]
好多地方都没有修改，点击files可以看到评论吗
image

**Comment by cccbbbttt** [comment]
修改了，请您看还有问题吗@ Tomoko-hjf

**Review by Fancy-hjyp — APPROVED** [review]
LGTM

**Comment by cccbbbttt** [comment]
@zhwesky2010 ，请问可以合并PR吗

**Comment by cccbbbttt** [comment]
@Tomoko-hjf，请问可以合并PR吗

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter_.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+                      index,
+                      src,
+                      reduce=None)
+```
+
+### [paddle.Tensor.put_along_axis_](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/put_along_axis__cn.html)
+
+```python
+paddle.Tensor.put_along_axis_(indices,
+                              value,
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter_.md:16]: API签名没写对，少个s

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter_add_.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+torch.Tensor.scatter_add_(dim,
+                         index,
+                         src)
+```
+
+### [paddle.Tensor.put_along_axis_](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/put_along_axis__cn.html)
+
+```python
+paddle.Tensor.put_along_axis_(indices,
+                              value,
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter_add_.md:15]: API签名没写对，少个s

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter_add_.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+其中 Paddle 相比 Pytorch 支持更多其他参数，具体如下：
+
+### 参数差异
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| dim           | axis        | 表示在哪一个维度 scatter ，仅参数名不一致。 |
+| index         | indices     | 表示输入的索引张量，仅参数名不一致。                   |
+| src           | values      | 表示需要插入的值，仅参数名不一致。                   |
+| -             | reduce      | 表示对输出 Tensor 的计算方式， PyTorch 无此参数, Paddle 应设置为 add 。  |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.scatter_add_.md:28]: 这两个API对比了结果没，应该不是完全一致的

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.scatter_add.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+其中 Paddle 相比 Pytorch 支持更多其他参数，具体如下：
+
+### 参数差异
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| input        | arr         | 表示输入 Tensor ，仅参数名不一致。                                     |
+| dim          | axis        | 表示在哪一个维度 scatter ，仅参数名不一致。                             |
+| index        | indices     | 表示输入的索引张量，仅参数名不一致。                                    |
+| src          | values      | 表示需要插入的值，仅参数名不一致。                                      |
+| -            | reduce      | 表示对输出 Tensor 的计算方式， PyTorch 无此参数, Paddle 应设置此参数为 add 。|
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.scatter_add.md:31]: 这两个映射对比下结果是否一致吧，put_along_axis可以映射到scatter，但是scatter_add可能还需要组合其他的逻辑

**Comment by Fancy-hjyp** [comment]
@cccbbbttt 同学你好，有时间的话可以按照修改意见完善一下映射文档~
